src/promptflow/promptflow/_sdk/_service/apis/run.py

```diff
@@ -220,3 +220,12 @@ class RestoreRun(Resource):
     def get(self, name: str):
         run = get_client_from_request().runs.restore(name=name)
         return jsonify(run._to_dict())
+
+
+@api.route("/<string:name>/delete")
+class DeleteRun(Resource):
+    @api.doc(description="Delete run")
+    @api.response(code=200, description="Delete run", model=dict_field)
+    def get(self, name: str):
```

Comment:
this should be a DELETE method, instead of GET?

Reply:
There ar4 multiple ways to make it.

Get /name/delete
Delete /name

I choose the first one.

Reply:
Another point is PFS azure delete is returning 204. I'll take some investigation on this.